artifacts/deploy/karmada-descheduler.yaml

```diff
@@ -24,7 +24,7 @@ spec:
           imagePullPolicy: IfNotPresent
           command:
             - /bin/karmada-descheduler
-            - --karmada-kubeconfig=/etc/kubeconfig
```

Comment:
Sorry, it was my mistake. Please use `--kubeconfig=/etc/kubeconfig`. 

Btw, there is another problem, I will test it again. This PR could be merged first.

Reply:
ok,thank you